numpy/random/mtrand.pyx

```diff
@@ -2137,6 +2137,9 @@ cdef class RandomState:
                Springer, 2002.
         .. [2] Wikipedia, "Student's t-distribution"
                https://en.wikipedia.org/wiki/Student's_t-distribution
+        .. [3] Walker, Helen M., "Degrees of freedom" in Journal of 
```

Comment:
No change to RandomState should be made.  It is in maintenance-only mode.

Reply:
Thank you for highlighting this, I was not aware. I will revert this file to its original state.

Reply:
Ehh, docstrings are a gray area that I didn't really consider in the NEP. We've allowed more significant code changes to `RandomState` since NEP 19. The core of the constraint in NEP 19 is that we don't change behaviors in `RandomState`, and this one doesn't.